utils/windows_configuration.py

```diff
@@ -37,8 +37,8 @@ def get_windows_sdk_check(name):
     try:
         with _winreg.OpenKey(_winreg.HKEY_LOCAL_MACHINE, sdk_reg_path) as reg_key:
             directory = _winreg.QueryValueEx(reg_key, "InstallPath")[0]
-            return (os.path.join(directory, 'Integrations', name, 'check.py'),
-                    os.path.join(directory, 'Integrations', name, 'manifest.json'))
+            return (os.path.join(directory, 'check.py'),
+                    os.path.join(directory,  'manifest.json'))
```

Comment:
Nit: there are two spaces between the comma and the "'": `directory,  'manifest.json'`